microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
 
+<!-- TODO: Redo screenshot -->
 ![The Crash analyzer tool](./index-images/crash-analyzer.png)
-<!-- todo: Create new demo page & use it in this article.
-Or, use page/file:
-https://devblogs.microsoft.com/oldnewthing/
-devtools://devtools/bundled/panels/elements/elements.js
-devtools://devtools/bundled/ui/components/tree_outline/tree_outline.js
 
-show stacktrace content in DevTools
--->
+When a web app crashes or hangs unexpectedly, this can result in data loss and a poor experience for your users.  Collecting information about the crashes that your users experience is helpful to diagnose and fix the root cause of the crashes. The **Crash analyzer** tool helps you map minified stack traces collected in production back to the original source code, so that you can quickly identify the root cause of the crashes.
 
-When a web app crashes or hangs unexpectedly, this can result in data loss and a poor user experience.  The **Crash analyzer** tool helps you identify the cause of these crashes.  Use the **Crash analyzer** tool to analyze the memory state of your web app at the time of a crash.
+There are two steps to analyze minified production stack traces in the **Crash analyzer** tool:
 
-As a web developer, you collect stack traces from customers in production, and then use the **Crash analyzer** tool to "beautify" the stack traces in DevTools.  DevTools only does the analysis of the stack traces, not the collecting of stack traces.
+1. First, you collect specially-formatted stack traces from your web app in production.
 
-When you select a crash from the list, you can see a detailed report of the crash, including:
-* The call stack.
-* The heap snapshot.
-* The exception message.
+   There are many ways to collect stack traces from the browsers your users are using. In [Collect stack traces in production](#collect-stack-traces-in-production), below, we provide examples and recommendations. We also explain how to format the stack traces so that they can be analyzed in the **Crash analyzer** tool.
 
-You can also inspect the values of the variables and objects in the heap, to see how they relate to the code that caused the crash.  This helps pinpoint the exact line of code that triggered the crash, and identify any memory leaks, infinite loops, or other bugs that may have contributed to it.
+1. Next, you analyze the stack traces in the **Crash analyzer** tool.
 
-The **Crash analyzer** tool helps debug and diagnose crashes that occur in any web app, regardless of the framework or library you use.  This tool can also help you test and optimize your web app for different browsers and devices, by comparing the crash reports from different scenarios and environments.
+   In [Analyze stack traces in the Crash analyzer tool](#analyze-stack-traces-in-the-crash-analyzer-tool), below, we explain how to first make source maps accessible in DevTools, and then how to use the **Crash analyzer** tool to analyze the stack traces.
```

Comment:
i'll try to reword to avoid "we".  see https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we
@captainbrosset

Reply:
fyi it was pretty straightforward to improve wording to move focus away from "we" to the doc'n structure:

The section [Analyze stack traces in the Crash analyzer tool](#analyze-stack-traces-in-the-crash-analyzer-tool), below, explains how to first make source maps accessible in DevTools, and then how to use the **Crash analyzer** tool to analyze the stack traces.

@captainbrosset